addons/portal/__manifest__.py

```diff
@@ -32,6 +32,10 @@
         'web._assets_frontend_helpers': [
             ('prepend', 'portal/static/src/scss/bootstrap_overridden.scss'),
         ],
+        'web.assets_backend': [
+            'portal/static/src/scss/portal_wizard.scss',
+            'portal/static/src/js/portal_wizard_user_field.js',
```

Comment:
Shouldn't it be in ``/views/fields``, as this seems to be the convention in owl code ?